manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md

```diff
@@ -10,30 +10,69 @@ products:
 
 # Configure a lifecycle policy [set-up-lifecycle-policy]
 
-For {{ilm-init}} to manage an index, a valid policy must be specified in the `index.lifecycle.name` index setting.
+An [{{ilm}}](/manage-data/lifecycle/index-lifecycle-management.md) ({{ilm-init}}) policy defines how your indices are managed over time, automating when and how they transition as they reach a certain age, size, or number of documents.
 
-To configure a lifecycle policy for [rolling indices](rollover.md), you create the policy and add it to the [index template](../../data-store/templates.md).
+There a few things to note about how an {{ilm-init}} policy works:
 
-To use a policy to manage an index that doesn’t roll over, you can specify a lifecycle policy when you create the index, or apply a policy directly to an existing index.
+* For {{ilm-init}} to manage an index, you need to specify a valid policy in the `index.lifecycle.name` index setting.
 
-{{ilm-init}} policies are stored in the global cluster state and can be included in snapshots by setting `include_global_state` to `true` when you [take the snapshot](../../../deploy-manage/tools/snapshot-and-restore/create-snapshots.md). When the snapshot is restored, all of the policies in the global state are restored and any local policies with the same names are overwritten.
+* To configure a lifecycle policy for [rolling indices](rollover.md) or data streams, you create the policy and add it to the [index template](../../data-store/templates.md).
 
-::::{important}
-When you enable {{ilm}} for {{beats}} or the {{ls}} {{es}} output plugin, the necessary policies and configuration changes are applied automatically. You can modify the default policies, but you do not need to explicitly configure a policy or bootstrap an initial index.
-::::
+* To use a policy to manage a single index, you can specify a lifecycle policy when you create the index, or apply a policy directly to an existing index.
```

Comment:
apply to existing indices link [doc](https://www.elastic.co/docs/manage-data/lifecycle/index-lifecycle-management/manage-existing-indices#ilm-existing-indices-apply)

Reply:
In that document you link to, the first two links point back to this "Configure a lifecycle policy" page. We don't want to send users into a circle of links.

---

<img width="618" alt="Screenshot 2025-06-27 at 1 24 51 PM" src="https://github.com/user-attachments/assets/09d32291-8817-4bfe-8a4b-95b4ef55c8ab" />